systemtest/driver/events.py

```diff
@@ -0,0 +1,16 @@
+import xml.etree.cElementTree as ET
+from pathlib import Path
+
+
+def read_starting_times(events_file: Path):
+    tree = ET.parse(events_file)
+    root = tree.getroot()
+    starting_times = {}
+    for event in root.iter("event"):
+        if event.attrib["state"] == "open" or event.attrib["state"] == "reset":
+            starting_times.setdefault(int(event.attrib["id"]), []).append(
+                float(event.attrib["time"])
+            )
+
+    assert starting_times, "Could not read starting times from events file"
```

Comment:
I strongly prefer if only tests and 'check' functions assert. Imo this should either throw or return 'None'.

Reply:
done